modules/react/button/lib/TertiaryButton.tsx

```diff
@@ -41,18 +37,15 @@ export interface TertiaryButtonProps extends Themeable, BaseButtonProps {
    */
   /**
    * If set to `true`, transform the icon's x-axis to mirror the graphic
-   * @default false
```

Comment:
Same issue as [above with PrimaryButton](https://github.com/Workday/canvas-kit/pull/1962#discussion_r1117789392) where the default for `shouldMirrorIcon` isn't showing up in TertiaryButton's prop table.

Reply:
I think this is a bug of the default parser. Probably a falsey check